packages/logging_cloudwatch/amplify_logging_cloudwatch/lib/src/queued_item_store/index_db/indexed_db_adapter.dart

```diff
@@ -138,15 +138,14 @@ class IndexedDbAdapter implements QueuedItemStore {
     await request.future;
 
     for (final elem in request.result) {
-      final value = elem as Map<String, dynamic>;
-      final id = value['id'] as int;
-      final itemValue = value['value'] as String;
-      final timestamp = value['timestamp'] as String;
+      final value = elem as Object;
```

Comment:
Also, why are we treating value as Object now?  We should prefer Dart's typing safety instead of casting objects to Object or dynamic.

Reply:
not casting as Object throws exception.

Reply:
What exception do you get though?  Casting to the wrong type will get an exception but doesn't mean we should cast to Object.

Reply:
it was casting to Map<String, dynamic> and the exception was something like `javascript object is not Map ...`. 
updated the code to call getCount method to reuse the implementation as you suggested.

Reply:
Yeah it's tricky with JSInterop.  I think you need to call '`SON.decode(` method to convert from a javascript object to a map.  Better would be to use codegen described here: https://github.com/aws-amplify/amplify-flutter/pull/3961#issuecomment-1767296183